### DISCUSSION TITLE
Reset pswrd

### DISCUSSION THREAD

**Discussion by adriangman** [dnschneid/crouton#4407] [Q&A] [answered]
When I press Ctrl+Alt+F2, I go into a login page but I forgot my password. How do I reset it?

**Comment by DennisLfromGA** [comment, marked as answer]
@adriangman,

If you have set a chronos and/or root password and cannot login and cannot remember the password then a powerwash will be needed to circumvent it which will wipe all accounts and user data, unfortunately.

However, logging in to the Developer Console via <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>F2</kbd> is *not* the recommended way to launch a crouton chroot. It is best done by signing-in to the account that you created your chroot(s)in then pressing <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>T</kbd> to get a crosh prompt then entering `shell` to get a bash prompt and launch your chroot(s) from there.

*Hope this helps,*
-**DennisLfromGA**

**Reply by Dozer257** [reply]
Yeah, you can't even launch a chroot from the Dev Console.
